api/app.go

```diff
@@ -215,25 +215,31 @@ func appList(w http.ResponseWriter, r *http.Request, t auth.Token) error {
 		w.WriteHeader(http.StatusNoContent)
 		return nil
 	}
+	simple, _ := strconv.ParseBool(r.URL.Query().Get("simplified"))
 	w.Header().Set("Content-Type", "application/json")
-	simple, _ := strconv.ParseBool(r.URL.Query().Get("nameOnly"))
+	miniApps := make([]miniApp, len(apps))
 	if simple {
-		appsNames := make([]interface{}, len(apps))
-		for i, appName := range apps {
-			res := map[string]interface{}{
-				"name": appName.Name,
+		for i, app := range apps {
+			ma := miniApp{
```

Comment:
I think we can use `minifyApp` here, passing a AppUnitsResponse with `.Error = nil` and `.Units = nil`.